spec/forms/admin/adjustments/create_adjustment_form_spec.rb

```diff
@@ -3,31 +3,28 @@
 require "rails_helper"
 
 RSpec.describe Admin::Adjustments::CreateAdjustmentForm, type: :model do
-  subject(:form) { described_class.new(created_adjustment_ids:, statement:, description:, amount:) }
+  let(:form) { described_class.new(created_adjustment_ids:, statement:, description:, amount:) }
```

Comment:
I see no advantage to this

the existing version means you can add a short `it { is_expected...... }` spec, the second doesn't ?

Reply:
no - for each of the describes below, the subject is set correctly to `form.save_adjustment`.
what I should do, is actually use `it { is_expected...... }` in those specs - will fix.